obspy/io/kinemetrics/core.py

```diff
@@ -66,9 +66,12 @@ def read_evt(filename_or_object, **kwargs):
 
     :type filename_or_object: str or file-like object
     :param filename_or_object: Evt file to be read
+    :type raw: bool
+    :param raw: if True, returns the raw data as saved in the Evt file, if
+        False (by default) returns the data converted to m/s**2.
```

Comment:
@ThomasLecocq Can you please check if it is viable to control this via `apply_calib` on `core.read()`? Whenever possible we should try to go with a unified approach.. (although that might mean that we can't keep the current default of converting by default for evt)

Reply:
@grapagnani @megies I think this makes sense if we default ``apply_calib`` to ``True``

Reply:
The default is `apply_calib=False` though in `Stream.read()`.. so I guess we can't combine it without a potentially breaking change.. I guess it might have to stay separate options then?

Reply:
I dunno what other formats use the ``apply_calib`` kwarg, it's one of those left-over kwargs from the past?

Reply:
Personally. I found odd the calibration was applied by default. It would have been a saner  decision to provide raw data as a default. I would be in favor of using `apply_calib` argument,  False by default. However it will be a breaking change. Maybe the  code could issue a python warning about this behavior change ?

Reply:
Yes, I did not notice when evt was first implemented that it was corrected by default, which we are not doing in other plugins, otherwise I would have stepped in back then. That correction that is being done, that is basically the same you would get from raw data + `remove_response()` to acceleration with the station metadata? If so, we should maybe go for a breaking change (and warn when the main `read()` function is used) to not correct by default anymore.

@ThomasLecocq yes `apply_calib` only ever was used in GSE2 I think, IIRC the reason was that the calibration factor in the waveform files was sometimes used complementary to the separately held station metadata..

Reply:
@megies I am not sure it is the same as raw data+remove_response(). The code applies a constant  conversion factor to the raw data. It may be sufficient for our particular case where the accelerometer has a flat response down to DC and the digitizer is sampling at max frequency of 250 Hz. However the EVT format is still used in recent Kinemetrics digitizer. When a seismometer is used with such digitizer, applying a constant factor will not be satisfactory, then raw data will be required.

Reply:
@ThomasLecocq @grapagnani in any case, we (pretty much) *always* just read the raw data, leaving conversion to physical units up to the user and station metadata. We really should consider doing the same for EVT, even though it is unfortunate that this will be a breaking change. But I would argue that consistency trumps backwards stability, open for other opinions though. If the waveforms have that flat correction factor encoded with the time series, I guess offering/advertizing the `apply_calib` switch with default `False` in this case (in the `io/kinemetrics/__init__.py` module docstring) might be the way to go?

Reply:
I agree. @grapagnani if you too, I'll adapt the code with a warning that should stay there until 2.1.0